content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/troubleshooting-tips.md

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
can you remove this file from the PR?

Reply:
yikes! yes. will try to be more careful with this moving forward